pgcli/key_bindings.py

```diff
@@ -102,9 +102,11 @@ def _(event):
     # history search, and one of several conditions are True
     @kb.add(
         "enter",
-        filter=~(has_completions | is_searching) & buffer_should_be_handled(pgcli),
+        filter=~(completion_is_selected | is_searching)
+        & buffer_should_be_handled(pgcli),
     )
     def _(event):
+        _logger.debug("Detected enter key.")
```

Comment:
I see the same debug line a few lines above - should we make the output distinct (maybe add "- handling buffer" or similar)?